pkg/proc/proc_test.go

```diff
@@ -3169,6 +3169,7 @@ func TestDebugStripped(t *testing.T) {
 	// Currently only implemented for Linux ELF executables.
 	// TODO(derekparker): Add support for Mach-O and PE.
 	skipUnlessOn(t, "linux only", "linux")
+	skipOn(t, "does not work with PIE", "pie")
```

Comment:
This test should work with PIE executables via the magic number lookup. I think for 1.21 the magic number lookup function just needs to be updated. I think instead of using `strings.Compare` we should use our own `AfterOrEqual`.